_includes/nav.html

```diff
@@ -6,12 +6,15 @@
 {% endfor %}
 {% endcapture %}
 
-<div class = 'navbar w3-bar' style='text-align: center'>
-    <a class = 'nav-button w3-bar-block w3-button' href="/">Home</a>
-    <span class = 'w3-hide-small'>
-        {{ page_list }}
-    </span>
-    <a href="javascript:void(0)" class="nav-button w3-bar-item w3-button w3-right w3-hide-large w3-hide-medium" onclick="toggleNav()">&#9776;</a>
+<div style='background-color: maroon'>
+    <h1 id = "title">FRC 6321 Scouting</h1>
+    <div class = 'navbar w3-bar' style='text-align: center'>
+        <a class = 'nav-button w3-bar-block w3-button' href="/">Home</a>
```

Comment:
This makes it take its own line (if this is intentional, ok, but a little unusual).

You could make it be part of the primary line with: 

```html
      <div class = 'navbar w3-bar' style='text-align: center'>
        <div id = "title" class="w3-bar-item">FRC 6321 Scouting</div>
        <a class = 'nav-button w3-bar-block w3-button' href="/">Home</a>
        <span class = 'w3-hide-small'>
...
```